src/index.js

```diff
@@ -5,7 +5,20 @@
  */
 
 const hammingDistance = (a, b) => {
-    
+  if (a.length != b.length) {
+    throw new Error('La longitud de los textos tiene que ser igual')
+  }else{
+    let texto1 = a.split('')
+    let texto2 = b.split('')
+    let numero = 0
+
+    for (let i = 0; i < a.length; i++) {
```

Comment:
Recuerda que los strings son iterables, puedes usar `for (i in a)` e `i` corresponde al índice de la posición en el string. Para acceder a la letra usas `a[i]`.